usr/src/uts/i86pc/sys/vmm.h

```diff
@@ -127,10 +127,30 @@ enum x2apic_state {
 #define	VM_INTINFO_HWEXCEPTION	(3 << 8)
 #define	VM_INTINFO_SWINTR	(4 << 8)
 
-
-#define	VM_MAX_NAMELEN	32
+/*
```

Comment:
When we go through vmadm we generate a name like `SYSbhyve-$ZONE_DID`, but anyone in the global zone can pass whatever name they would like to bhyve. I often spin up bhyve instances outside of a zone for debugging under mdb etc.  If we don't have a device name length limit like FreeBSD, should we just set `VM_MAX_NAMELEN` to some reasonable value?

Reply:
Ran grep over the illumos-joyent for VM_MAX_NAMELEN and boot.c is the only other place it came up because of the comment.  Decided to add an ifdef'd comment in the header where this calculation happens.